README.rst

```diff
@@ -69,10 +69,16 @@ Example usage::
 generates a file called ``migrations/20151217170514_add_id_to_users.py``
 with content::
 
+    # Uncomment should_run if this is a repeat migration
+    # def should_run(cursor):
```

Comment:
This is not a case for schema migrations, more for data migrations.  Please see the [original issue](Connexions/db-migrator#1).